### PR TITLE
Show users options to download packages rather than just .jar file.

### LINKED ISSUE #12: "Download" button on home page is effectively broken for some(or many or most?) users
## Current behavior

Clicking the "Download" button at the top of https://openrocket.info downloads the OpenRocket `.jar` file.  However, as noted on the [Releases page](https://github.com/openrocket/openrocket/releases)...
> If you have a more recent version [of Java] on your computer (as nearly everyone does), you will not be able to run the .jar file below without downgrading

**This caveat does not appear anywhere on the home page!**

OpenRocket will fail to run for users experiencing this problem, and will fail in incomprehensible ways, as evidenced by https://github.com/openrocket/openrocket/issues/759.

## Desired Behavior

For most users, the `.jar` file is (and should be) the option of last resort.  Windows and Mac users almost certainly prefer `.exe` or `.dmg` installations, respectively.  And Linux users may prefer `.AppImage` installations, although I'm not as familiar with that community so can't be sure about that.

Ideally the "Download" button would cause the appropriate installer (.exe, .dmg, .AppImage) to download based on the user's OS.  Failing that, taking the user's to the Github Releases page where they will at least be apprised of the `.jar` issue, and can select the appropriate installer for themselves will avoid this unpleasantness.

### PR DESCRIPTION
**Pull request by JoePfeiffer**
Modify packages.md to give list of packaging options, and delete references
to Android version

Modify download_button.html to point to packages.md instead of .jar file.

Closes #12

### CODE REVIEW

**Comment by neilweinstock** [comment]
Is there a way to preview the updated HTML in a browser, or can I only review the source changes?

The text mostly looks good: the one addition needed is for the Windows version.  The crappy installer (assuming it's the one I built a million years ago) doesn't create the usual shortcuts on the desktop or taskbar, so you have to do it manually.  In my thread on TRF, I wrote:

"The application uses a standard installer, but it doesn't (at present) provide a shortcut on the desktop or the start menu. After installing, the actual executable file for the application is installed into your user/AppData/Local/OpenRocket folder. So you can go in there, right click the "OpenRocket.exe" file, and create a shortcut, pin it to the taskbar and/or pin it in the start menu, whatever."

I think you should include a note to that effect (hopefully better written than my text above), since otherwise it's a bit bewildering that after the install you can't find the program.

**Comment by JoePfeiffer** [comment]
As usual, I only know how to do this under Linux.  I assume there is some sort of similar mechanism under macOS and Windows.

Anyway, under Linux:

1. clone or download the repo as usual
2. install the jekyll web page generator.  jekyll is the only program I have on my Debian system written in Ruby, and a simple 'sudo apt-get jekyll' succeeded in pulling in Ruby and all the gems needed.
3. go to the top level directory of the local copy and run 'jekyll serve'
4. this will serve up the pages on localhost:4000

So far as your text is concerned, it seems fine to me except that I'd like to delete ', whatever.'  I can add that and push it if you'd like.

**Comment by neilweinstock** [comment]
Sure you can add my text.

**Comment by wolsen** [comment]
Overall, this looks fine to me. I'll wait for an update from Joe before merging though

File: packages.md
Comment (wolsen) [inline packages.md:27]: As an FYI, users will also be able to use the snap installation from the snap store on linux. Its available now, but it delivers the unstable branch so I think we should hold off on mentioning it here. Just a note for future self more than anything

Comment (JoePfeiffer) [reply packages.md:27]: Not quite sure why the lines of code quoted in your comment -- is there a suggested change in there?

Right, I don't think we want to announce the snap on our main download page until we're in beta.

**Review by neilweinstock — APPROVED** [review]

File: packages.md
Comment (wolsen) [reply packages.md:27]: uh, its just where I clicked in relation to the linux context. sorry for the confusion

**Pull request merged**
